index.html

```diff
@@ -48,6 +48,7 @@ <h2>Learn</h2>
 						<li><a href="docs/index.html#manual/introduction/Creating-a-scene">documentation</a></li>
 						<li><a href="examples/#webgl_animation_keyframes">examples</a></li>
 						<li><a href="editor/">editor</a></li>
+						<li><a href="playground/">playground</a></li>
```

Comment:
Should this be `node playground`?

Reply:
> Should this be `node playground`?

Done.

Reply:
@Mugen87 Maybe `node editor`?

Reply:
Yes, that's even better.

Reply:
330952bbc55016f2302ce595fb3c372b714a0f5f